docs/observability/README.md

```diff
@@ -24,6 +24,7 @@ Building observable systems enables development teams at CSE to measure how well
 - [Tooling and Patterns](tools/README.md)
 - [Observability As Code](observability-as-code.md)
 - [Recommended Practices](best-practices.md)
+- [Non-Functional Requiurements Guidance](../design/design-patterns/non-functional-requirements-investigation-guide.md)
```

Comment:
Typo "Requirements" and suggest this is added to a new section at the bottom of the page called "Useful Links" vs in the main structure of the Observability section as this reflects the structure of observability itself.

Reply:
Done,